doc/general_documentation/flight_release.md

```diff
@@ -0,0 +1,89 @@
+\page release Creating a Flight Release
+
+# Creating a Flight Release
+
+This page describes the steps required to complete a flight release. It assumes
+that your environment is configured to cross-compile the fsw for the `armhf`
+architecture.
+
+## Create a Release Branch
+
+Release branches will be created on a user's fork. To make a release branch,
+checkout and update develop. Then make a release branch off of develop.
+
+    git checkout develop
+    git fetch upstream
+    git merge upstream/develop
+    git checkout -b release-0.x.x
+
+
+## Update the Release Version
+
+The release version must be updated before creating the Debian. Please use
+the update release script to do this. Before running the script, get a list of
+features that have been added since the last release. This list will need to be
+added to the change log that gets edited when the script is run. Also, in the
+change log, change UNRELEASED to testing.
```

Comment:
and change the email address to the FSW email address in the previous log entries